crates/project_model/src/build_scripts.rs

```diff
@@ -94,17 +94,17 @@ impl WorkspaceBuildScripts {
             by_id.insert(workspace[package].id.clone(), package);
         }
 
-        let mut callback_err = None;
```

Comment:
We were never using this "error case" here, but once set we stop working through the build script output, so this might be the cause of https://github.com/rust-analyzer/rust-analyzer/issues/9720 not working for people without any apparent reason?